docs/visualizing-your-sql-queries/plotting-with-seaborn.md

```diff
@@ -1,14 +1,559 @@
+
+<b>Hint</b> Try to find an example that does exactly what the question asks in the [documentation](https://seaborn.pydata.org/generated/seaborn.relplot.html)
+
+<!-- #region -->
+<details>
+
+<summary>Answers</summary>
+
+We first modify the `relplot_example` CTE by removing the filter for `duration`:
+
+```{code-cell} ipython3
+%%sql --save relplot_question
+SELECT status, duration, amount, region, unemployment_rate_96
+FROM s1.account AS a
+LEFT JOIN s1.loan AS l
+ON a.account_id = l.account_id
+LEFT JOIN s1.district AS d
+ON d.district_id = a.district_id
+WHERE region IN ('north Moravia', 'south Moravia') AND
+status IN ('A', 'C');
+```
+
+Save the CTE as a pandas `DataFrame()`:
+
+```{code-cell} ipython3
+result = %sql SELECT * FROM relplot_question
+df = result.DataFrame()
+```
+
+Finally, `sns.relplot` is called and stored in a variable for customizing the x-axis label. Note that the `sizes` argument specifies magnitude of the point `size`, which is used to control the visibility of the points:
+
+```{code-cell} ipython3
+g = sns.relplot(
+    data=df,
+    x="unemployment_rate_96",
+    y="amount",
+    size="duration",
+    style="region",
+    hue="status",
+    col="status",
+    palette=["b", "r"],
+    sizes=(10, 100),
+)
+g.set(xlabel=None, ylabel="Loan Amount ($)")  # remove duplicate x axis label
+g.fig.supxlabel("Unemployment Rate (%)")  # x-axis label
+plt.show(g)
+```
+
+</details>
+<!-- #endregion -->
+
+## Density plots
+
+A kernel density estimate (KDE) plot is a method for visualizing the distribution of observations in a dataset, analogous to a histogram. KDE represents the data using a continuous probability density curve in one or more dimensions. Relative to a histogram, KDE can produce a plot that is less cluttered and more interpretable, especially when drawing multiple distributions.
+
+```{important}
+The bandwidth, or standard deviation of the smoothing kernel, is an important parameter. Misspecification of the bandwidth can produce a distorted representation of the data.
+```
+
+### Example
+
+Seaborn's [`kdeplot` <b>Axes-level function</b>](https://seaborn.pydata.org/generated/seaborn.kdeplot.html) can help us easily visualize KDE's of multiple numeric variables. Its Figure-level equivalent is the [`displot` function](https://seaborn.pydata.org/generated/seaborn.displot.html#seaborn.displot) with which we can produce KDE plots by specifying `kind="kde"`.
```

Comment:
Axis singular, axes plural

Reply:
axes is appropriate, shall fix capitalization